### ISSUE TITLE
Add support for Write-Progress with integrated console

### ISSUE THREAD

**Issue by jdhitsolutions** [PowerShell/PowerShellEditorServices#384]
Greatly appreciate the PowerShell integrated console introduced in 0.10.0. It would be helpful to include support for Write-Progress.  Currently code like this:
```
1..100| foreach { write-progress -Activity Working -Status Counting -CurrentOperation $_; start-sleep -Milliseconds 100}
```
Runs without error but also without any display.

**Comment by jdhitsolutions** [comment]
Not sure if this belongs here or https://github.com/PowerShell/vscode-powershell/issues

**Comment by dragonwolf83** [comment]
I vote for making it show up in console first. Later, it would be great to have a setting to allow for an Editor UI to override it.

With ISE, progress looks nice for one item, but was a bit too heavy when doing multiple progress bars. IIRC, the Console had a better experience and I was actually wanting to turn off the UI after I saw that.

**Comment by SeeminglyScience** [comment]
Oops this should have been closed ages ago with #874
